site/cds_rdm/assets/semantic-ui/js/cds_rdm/src/records/detail.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2016-2022 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import $ from "jquery";
+
+// Initialize conceptdoi modal
+$("#record-conceptdoi-badge").on("click", function () {
```

Comment:
shelve: ideally the event should be generic for any modal, not depending on the element ID

Reply:
I am not sure that I understand your comment, we want to open that exact modal, no?

Reply:
yes, only the one specific modal should be opened, but it is possible to implement in a generic way (if not already done in the plain semantic)